middleware/apimiddleware.go

```diff
@@ -679,7 +679,7 @@ func BlockIndexLatestCtx(r *http.Request, source DataSource) context.Context {
 }
 
 // StatusCtx embeds the specified apitypes.Status into a request context.
-func StatusCtx(r *http.Request, status apitypes.Status) context.Context {
+func StatusCtx(r *http.Request, status *apitypes.Status) context.Context {
```

Comment:
`m.StatusCtx` isn't actually used anywhere, but now that `apitypes.Status` embeds a mutex, I could no longer pass it by value.